### PR TITLE
Robrap test broken test

### PR DESCRIPTION
**Pull request by robrap**

### CODE REVIEW

**Comment by feanil** [comment]
jenkins run microsites

File: regression/pages/whitelabel/logout_page.py [hunk 1]
```diff
@@ -3,34 +3,29 @@
 """
```
Comment (robrap) [inline regression/pages/whitelabel/logout_page.py:3]: Take all changes from  regression/pages/whitelabel/logout_page.py.

File: regression/tests/whitelabel/course_enrollment_test.py [hunk 2]
```diff
@@ -98,13 +98,13 @@ def verify_price_on_basket(self):
...
 
-    def assert_enrollment_and_logout(self):
+    def assert_enrollment_and_logout_of_ecommerce(self):
         """
         Verify that course is added to user dashboard and user can access
         the course. After that, logout from application.
         """
         self.assert_course_added_to_dashboard()
-        self.logout_from_ecommerce_using_api()
+        self.logout_user_from_ecommerce()
```
Comment (robrap) [inline regression/tests/whitelabel/course_enrollment_test.py:107]: Keep, and ensure rename happens globally.

File: regression/tests/whitelabel/test_discount_coupon.py [hunk 3]
```diff
@@ -163,90 +169,108 @@ def test_discount_single_use_fixed_redeem_url(self):
...
-        self.single_seat_basket.wait_for_page()
-        self.make_payment_after_discount()
-        self.assert_course_added_to_dashboard()
-        redeem_coupon = RedeemCouponPage(self.browser, coupon_code).visit()
-        self.assertEqual(
-            redeem_coupon.error_message,
-            ONCE_PER_CUSTOMER_REDEEM_URL_SAME_USER_REUSE
-        )
+            log.error("Completed dashboard_page.wait_for_page()");
+            self.assert_enrollment_and_logout_of_ecommerce()
```
Comment (robrap) [inline regression/tests/whitelabel/test_discount_coupon.py:199]: Just need these renames.

File: regression/tests/whitelabel/voucher_tests_base.py [hunk 4]
```diff
@@ -107,17 +111,19 @@ def verify_receipt_info_for_discounted_course(self):
             self.receipt_page.order_date
         )
 
-    def redeem_single_course_discount_coupon(self, coupon_url):
+    def redeem_single_course_discount_coupon(self, coupon_code):
```
Comment (robrap) [inline regression/tests/whitelabel/voucher_tests_base.py:114]: Rename this variable.

File: regression/tests/whitelabel/voucher_tests_base.py [hunk 5]
```diff
@@ -107,17 +111,19 @@ def verify_receipt_info_for_discounted_course(self):
...
         """
         Redeem single course discount coupon.
 
         Arguments:
-            coupon_url: Url of the coupon.
+            coupon_code: Url of the coupon.
```
Comment (robrap) [inline regression/tests/whitelabel/voucher_tests_base.py:119]: Fix this comment.

File: regression/tests/whitelabel/white_label_tests_base.py [hunk 6]
```diff
@@ -67,7 +67,7 @@ def logout_user_from_ecommerce(self):
         """
         Logout user from ecommerce site
         """
-        self.logout_page.logout_from_ecommerce()
+        self.logout_page.visit()
```
Comment (feanil) [inline regression/tests/whitelabel/white_label_tests_base.py:70]: We'll want to pull in this change as well.

**Comment by robrap** [comment]
Closing in favor of this cleaned-up PR: https://github.com/edx/edx-e2e-tests/pull/401

**Pull request closed without merging**